packages/itwinui-css/backstop/tests/table.html

```diff
@@ -124,6 +124,16 @@
         }
       );
     </script>
+    <script>
```

Comment:
Just wondering, can we merge all the `customElements.define` code from the above lines into just one `<script>`? Or is there a reason they are all in separate tags?

Reply:
It can be done in the same script, but this is just the convention we've been following. No reason in particular.